src/appendices/qmri.md

```diff
@@ -133,10 +133,13 @@ A guide for using macros can be found at
 
 Please visit the [file collections appendix](./file-collections.md#magnetic-resonance-imaging) to see the list of currently supported qMRI applications.
 
-### Quantitative maps are derivatives
+### Outputs are quantitative maps
 
-Regardless of how they are obtained (pre- or post-generated), qMRI maps are stored in the `derivatives` directory.
-For example a `T1map` can be generated from an `MP2RAGE` file collection using either options.
+qMRI maps are stored differently depending on the process that generated them.
+Pre-generated qMRI maps MUST be stored as part of raw BIDS datasets,
```

Comment:
```suggestion
Pre-generated qMRI maps MAY be stored as part of raw BIDS datasets,
```

This is unenforceable and I don't think we should require it, anyway.

Somebody might prefer to create a dataset like:

```
derivatives/
  scanner-generated/
    sub-01/
      anat/
        sub-01_T1map.nii.gz
        sub-01_T1map.json
        sub-01_UNIT1.nii.gz
        sub-01_UNIT1.json
  open-algorithm/
    sub-01/
      anat/
        sub-01_T1map.nii.gz
        sub-01_T1map.json
        sub-01_UNIT1.nii.gz
        sub-01_UNIT1.json
sub-01/
  anat/
    sub-01_inv-1_part-mag_MP2RAGE.nii.gz
    sub-01_inv-1_part-phase_MP2RAGE.nii.gz
    sub-01_inv-1_MP2RAGE.json
    sub-01_inv-2_part-mag_MP2RAGE.nii.gz
    sub-01_inv-2_part-phase_MP2RAGE.nii.gz
    sub-01_inv-2_MP2RAGE.json
```

There the symmetry between the different derivatives would make the dataset more legible.